CodeGeneration/Sources/generate-swiftparser/templates/DeclarationAttributeFile.swift

```diff
@@ -0,0 +1,47 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import SyntaxSupport
+import Utils
+
+let declarationAttributeFile = SourceFile {
+  ImportDecl(
+    """
+    \(copyrightHeader)
+    @_spi(RawSyntax) import SwiftSyntax
+    
+    """
+  )
+  
+  ExtensionDecl("extension Parser") {
+    EnumDecl(
+      identifier: "DeclarationAttribute",
+      inheritanceClause: TypeInheritanceClause {
+        InheritedTypeSyntax(
+          typeName: SimpleTypeIdentifier(stringLiteral: "SyntaxText"),
+          trailingComma: TokenSyntax.comma)
+        
+        InheritedTypeSyntax(
+          typeName: SimpleTypeIdentifier(stringLiteral: "ContextualKeywords"),
+          trailingComma: nil)
+      },
+      membersBuilder: {
```

Comment:
Can’t this be simplified to 

```suggestion
    EnumDecl("enum DeclarationAttribute: SyntaxText, ContextualKeyword") {
```